app/views/insured/fdsh_ridp_verifications/failed_validation.html.erb

```diff
@@ -10,7 +10,7 @@
     <%= render partial: 'notices/shared/unordered_list', locals: { previous: l10n('insured.interactive_identity_verifications.failed_validation.else'), listings: [l10n('insured.interactive_identity_verifications.failed_validation.step_1'), l10n('insured.interactive_identity_verifications.failed_validation.step_2', transaction_id: @verification_transaction_id), l10n('insured.interactive_identity_verifications.failed_validation.step_3')] } %>
     <% unless @verification_transaction_id.blank? %>
       <p><%= l10n("insured.interactive_identity_verifications.failed_validation.if_continue") %></p>
-      <p class="mb-4"><%= sanitize(link_to l10n("insured.consumer_roles.upload_ridp_documents.documents_faq"), ::EnrollRegistry[:enroll_app].setting(:submit_docs_url).item, class: "button outline", target: '_blank', rel: 'noopener noreferrer') %> <%= sanitize(link_to(l10n('insured.interactive_identity_verifications.failed_validation.continue_button'), insured_interactive_identity_verification_path(:id => @verification_transaction_id), :method => :put, :class => "btn-secondary interaction-click-control-override-identity-verification" + pundit_class(Family, :updateable?))) %></p>
+      <p class="mb-4"><%= sanitize(link_to l10n("insured.consumer_roles.upload_ridp_documents.documents_faq"), ::EnrollRegistry[:enroll_app].setting(:submit_docs_url).item, target: '_blank', class: "button outline", rel: 'noopener noreferrer') %> <%= sanitize(link_to(l10n('insured.interactive_identity_verifications.failed_validation.continue_button'), insured_interactive_identity_verification_path(:id => @verification_transaction_id), :method => :put, :class => "btn-secondary interaction-click-control-override-identity-verification" + pundit_class(Family, :updateable?))) %></p>
```

Comment:
the issue here is the `sanitize` which removes html attributes includ target blank. this should be switched to a h instead, so like `<%= h(link_to l10n("insured.consumer_roles.upload_ridp_documents.documents_faq"), ::EnrollRegistry[:enroll_app].setting(:submit_docs_url).item, target: '_blank', class: "button outline", rel: 'noopener noreferrer') %>`